projects/m1/010-month-name-to-number-of-days/solutions/Jurgens Marpali/month-to-number.py

```diff
@@ -0,0 +1,22 @@
+#The length of a month varies from 28 to 31 days. 
+#In this exercise you will create a program that reads the name of a month from the user as a string. 
+#Then your program should display the number of days in that month. Display “28 or 29 days” for February so
+#that leap years are addressed.
+month_31 = ('january', 'march', 'may', 'july', 'august', 'october', 'december')
```

Comment:
Actually, you don't need to define all the case.

For example, if a month is not composed by 28/29/30 days, it will fall in the 31 case.

What do you think?

You can improve this and saving work to your code

Reply:
I think that you are right, but i tried not to define the 31 days long months and the code will not act as i want it to act when i type in "february" in the console. It will first tell me that it has 31 days and then it will ask me the year so he can proceed to tell me if it's a leap year or not.